packages/cf-style-const/src/variables.js

```diff
@@ -97,5 +97,5 @@ export default {
   colorFacebookBlue: '#3b5998',
   colorMainBackground: '#e6e6e6',
   zIndexMax: 1000,
-  modalBoxShadow: '0 0 0 9999px rgba(0, 0, 0, .7)'
+  modalBackdropBackground: 'rgba(0, 0, 0, .7)'
```

Comment:
@jwineman Not sure I understand, you want `rgba(0,0,0,0.7)` as a generic color const that's unrelated to modals?

Maybe something like overlayColor: 'rgba(0,0,0,0.7)` ?